File: examples/example-43/js/faker.js
const { faker } = require('@faker-js/faker');

function generateUserAccount() {
    const name = faker.name.findName();
    const city = faker.address.city();
    const state = faker.address.state();
    const phoneNumber = faker.phone.phoneNumber();

    const emailLocalPart = name.toLowerCase()
        .replace(/\s+/g, '.')
        .replace(/[^a-z0-9.]/g, '');
    const email = `${emailLocalPart}@sink.mywebsite.com`;

    return { name, email, city, state, phoneNumber };
}

module.exports = { generateUserAccount };

File: examples/example-43/ruby/faker.rb
require 'faker'

def generate_user_account
    name = Faker::Name.name
    city = Faker::Address.city
    state = Faker::Address.state
    phone_number = Faker::PhoneNumber.phone_number
    email_local = name.downcase.gsub(/\s+/, '.').gsub(/[^a-z0-9.]/, '')
    email_address = "#{email_local}@sink.mywebsite.com"

    {
        name: name,
        email: email_address,
        city: city,
        state: state,
        phone_number: phone_number
    }
end